packages/sources/css/src/components/indicators/tag/src/index.css

```diff
@@ -94,3 +94,19 @@ a.vtmn-tag:focus-visible {
   background-color: var(--vtmn-semantic-color_decorative-emerald);
   color: var(--vtmn-semantic-color_content-primary);
 }
+
+/* SIZES */
+
+/* Small */
```

Comment:
on the class `.vtmn-tag` you can remove this lines : 

```css
min-block-size: rem(24px);
block-size: rem(24px);
font-size: var(--vtmn-typo_text-3-font-size);
```

It's a duplicate of `.vtmn-tag_size--medium` :)